wordpress-mobile/WordPress-iOS
# Signup epilogue: show SVProgressHUD only for user updated display name

Fixes #8864 

To test:
- Signup with email.
- On the epilogue, don't change any values.
- Press 'Continue'.
- Verify the 'Changing display name' HUD is _not_ displayed.

- Signup with email.
- On the epilogue, change the display name.
- Press 'Continue'.
- Verify the HUD _is_ displayed.
